__tests__/components/CRTasks.test.ts

```diff
@@ -0,0 +1,225 @@
+import {
```

Comment:
I deport seem to see a test where the user is logged out, can you confirm ?, best